api/managementpb/actions.proto

```diff
@@ -55,13 +55,16 @@ message StartMySQLExplainActionRequest {
       string_not_empty: true
     }
   ];
-  // Query ID or query is required.
-  // Value of query.
-  string query = 3;
-  // Query ID of query. 
-  string query_id = 6;
+  // Deprecated: should not be used.
+  string query = 3 [deprecated = true];
+  // Query ID of query. Required. 
+  string query_id = 6 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
```

Comment:
let's keep it optional for a few versions, we have to support backward compatibility with old pmm-agents

Reply:
Added back query for MySQL explain actions. Just marked "deprecated" by comment. Query or query ID required. Because if query ID should be optional we need also query.